src/components/pages/PortfolioManagement/PortfolioManagement.tsx

```diff
@@ -263,33 +265,36 @@ export const PortfolioManagement = () => {
             {userDelistedCurrenciesArray.length > 0 && (
                 <div className='px-3 laptop:px-0'>
                     <Alert
-                        severity='error'
-                        showCloseButton={true}
+                        severity={AlertSeverity.Error}
                         localStorageKey={DELISTED_CURRENCIES_KEY}
                         localStorageValue={Array.from(delistedCurrencySet)
                             .sort()
                             .join()}
-                    >
-                        <p className='text-white'>
-                            Please note that your contracts for{' '}
-                            {generateDelistedCurrencyText(
-                                userDelistedCurrenciesArray
-                            )}{' '}
-                            will be delisted at maturity on Secured Finance.{' '}
-                            <a
-                                className='whitespace-nowrap text-secondary7 underline'
-                                href='https://docs.secured.finance/product-guide/loan-market-platform/loan-assets/listing-and-delisting'
-                                target='_blank'
-                                rel='noreferrer'
-                                onClick={e => {
-                                    e.preventDefault();
-                                    scrollToBottom();
-                                }}
-                            >
-                                Learn more
-                            </a>
-                        </p>
-                    </Alert>
+                        title={
+                            <>
+                                <p className='text-white'>
```

Comment:
is it intentional to pass the text color here?

Reply:
I've removed the old implementation now. Since the Alert handles text color with neutral-50 I'll remove text-white here